client/ayon_core/lib/attribute_definitions.py

```diff
@@ -6,82 +6,53 @@
 import copy
 import warnings
 from abc import ABCMeta, abstractmethod
-from typing import Any, Optional
+import typing
+from typing import (
+    Any,
+    Optional,
+    List,
+    Set,
+    Dict,
+    Iterable,
+    TypedDict,
```

Comment:
TypedDict is Py3.8+ so adding this drops support for Py3.7

Reply:
Do we support 3.7 now?